tests/x11/reboot_gnome.pm

```diff
@@ -16,11 +16,26 @@ use strict;
 use testapi;
 use utils;
 
+sub assert_and_click_until_screen_change {
+    my ($mustmatch, $wait_change, $repeat) = @_;
+    $wait_change //= 1;
```

Comment:
and also it will most likely not do what is expected because @SergioAtSUSE uses it as parameter on `wait_screen_change`, not `assert_and_click`. Just delete that parameter because you don't need it anymore.